app/views/guides/pages/getting-started/configuration.haml

```diff
@@ -81,12 +81,19 @@
     %tr
       %td
         %code application.icon
-        %td A path to an image file which is converted to different icons like [favicons](https://en.wikipedia.org/wiki/Favicon).
+        %td
+          A path to an image file which is converted to different icons like
+          %a(href="https://en.wikipedia.org/wiki/Favicon")
+            favicons
+          .
     %tr
       %td
         %code application.orientation
         %td
-          [PWA] The orientation of the application, either
+          [
+          %a(href="https://en.wikipedia.org/wiki/Progressive_web_app")
+            PWA
+          ] The orientation of the application, either
```

Comment:
I'd leave this one alone, this term should be explained beforehand.

Reply:
Makes sense. But I will not try to explain it, as it is new to me.